backend/forecastbox/config.py

```diff
@@ -82,7 +108,11 @@ class CascadeSettings(BaseModel):
 class FIABConfig(BaseSettings):
     model_config = SettingsConfigDict(env_file=".env", env_nested_delimiter="__", env_prefix="fiab__")
 
+    url: str
```

Comment:
which url is it? Frontend? Let's name it `frontend_url`, to distinguish from `api.api_url`?